packages/datepickers/src/styled/StyledHeaderPaddle.ts

```diff
@@ -6,37 +6,63 @@
  */
 
 import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
-import { retrieveComponentStyles, getColorV8, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
 
-const retrieveSizing = ({
-  isCompact,
-  theme
-}: { isCompact?: boolean } & ThemeProps<DefaultTheme>) => {
+const sizeStyles = ({ $isCompact, theme }: { $isCompact?: boolean } & ThemeProps<DefaultTheme>) => {
   let size = theme.space.base * 10;
 
-  if (isCompact) {
+  if ($isCompact) {
     size = theme.space.base * 8;
   }
 
   return css`
     width: ${size}px;
     height: ${size}px;
+
+    svg {
+      width: ${props => `${props.theme.iconSizes.md}`};
+      height: ${props => `${props.theme.iconSizes.md}`};
```

Comment:
Move to a local `iconSize` variable rather than invoking the `props =>` function.